applications/FluidDynamicsApplication/tests/cpp_tests/test_embedded_fluid_element.cpp

```diff
@@ -0,0 +1,259 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Jordi Cotela
+//                   Ruben Zorrilla
+//
+
+#include <iomanip> // for std::setprecision
+
+// Project includes
+#include "testing/testing.h"
+#include "includes/model_part.h"
+#include "includes/cfd_variables.h"
+
+// Application includes
+#include "custom_constitutive/newtonian_2d_law.h"
+
+namespace Kratos {
+namespace Testing {
+
+KRATOS_TEST_CASE_IN_SUITE(EmbeddedElement2D3N, FluidDynamicsApplicationFastSuite)
+{
+    ModelPart model_part("Main");
+    model_part.SetBufferSize(3);
+
+    // Variables addition
+    model_part.AddNodalSolutionStepVariable(DENSITY); // TODO: To be removed once the element migration is finally finished
+    model_part.AddNodalSolutionStepVariable(DYNAMIC_VISCOSITY); // TODO: To be removed once the element migration is finally finished
```

Comment:
Can we remove these two in the test? Do the navier_stokes elements use a constitutive law?